src/Runner.Worker/FileCommandManager.cs

```diff
@@ -208,9 +210,24 @@ public void ProcessCommand(IExecutionContext context, string filePath, Container
                     ? context.Id.ToString() 
                     : context.EmbeddedId.ToString();
 
-                Trace.Info($"Queueing file ({filePath}) for attachment upload ({attachmentName})");
-                // Attachments must be added to the parent context (job), not the current context (step)
-                context.Root.QueueAttachFile(ChecksAttachmentType.StepSummary, attachmentName, scrubbedFilePath);
+                var resultsReceiverEndpoint = context.GetGitHubContext("resultsReceiverEndpoint");
+                if (resultsReceiverEndpoint != null) 
+                {
+                    Trace.Info($"Got results receiver endpoint: {resultsReceiverEndpoint}");
+                    Trace.Info($"Queueing results file ({filePath}) for attachment upload ({attachmentName})");
+                    // TODO: Determine if this is the actual access token from the token service
```

Comment:
@TingluoHuang I think this is one of the last things we are struggling with (other than actually getting everything working end-to-end). Is this where the accesstoken lives that is obtained from the actions service? If not, where do we find that...? It should already have the correct scopes added to it to reach out to the results service